server/src/main/java/io/mantisrx/server/master/store/MantisJobMetadataWritable.java

```diff
@@ -86,9 +87,8 @@ public MantisJobMetadataWritable(@JsonProperty("jobId") String jobId,
         this.name = name;
         this.user = user;
         this.submittedAt = submittedAt;
-        if(startedAt == 0) {
-            this.startedAt = submittedAt;
-        }
+        this.startedAt = startedAt;
```

Comment:
should this be equal to `submittedAt` if not set ?

Reply:
In the Job Actor `setRuntimeLimitTimersIfRequired()` method if the startedAt time was not set it will be overridden to use submittedAt time (as it is guaranteed that this method is called only for a running job). In the POJO itself it could be legit to not have startedAt set as the job may not have entered into the running state.

Reply:
sounds good.